recipes/editors/24.7/environments.yaml

```
@@ -0,0 +1,30 @@
+editors:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  unify: true
+  mpi: null
+  specs:
+  # the editors
+  - neovim
+  - emacs +treesitter
+  - nano
+  - vim +python +lua
+  # additional tools that integrate well with tools
+  - lua
+  - node-js
+  - python@3.12
+  - rust
+  - ripgrep
```

```suggestion
  - ripgrep
  - fd
  - lazygit
```
`fd` requested by @simonpintarelli

---

added